server.py

```diff
@@ -28,6 +31,13 @@ def updateBoard(self,imagedata):
 }
 server = Server(imagedata)
 
+@socketio.on('connect', namespace='/canvas')
+def connect_canvas():
+    # broadcast board upon initial connect at /canvas endpoint
+    # turn board into JSON
+    board = CanvasBoardEncoder().encode(server.board)
+    emit('broadcast-board', board)
+
 @socketio.on('connect')
 def on_connect():
     print("connected to websocket")

```

Comment:
Do we still need this?

Reply:
Removed unnecessary on_connect() since websocket endpoint is changed from '/' to '/canvas'